### ISSUE TITLE
MD028 flags consecutive quotes

### ISSUE THREAD

**Issue by zane** [rvben/rumdl#126]
## Reproduction steps

```
rumdl check - <<EOF
# Heading

> Hello.

> Goodbye.
EOF
```

## Expected results

```
No issues found in <stdin>
```

## Actual results

```
<stdin>:4:1: [MD028] Blank line inside blockquote (level 1) [*]

Found 1 issue(s) in <stdin>
```

**Comment by rvben** [comment]
Hi @zane,

Thanks for raising this issue. While this is valid according to CommonMark, but as also the [markdownlint docs](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) explain; "Some Markdown parsers will treat two blockquotes separated by one or more blank lines as the same blockquote, while others will treat them as separate blockquotes." I have updated the [rumdl docs for MD028](https://github.com/rvben/rumdl/blob/main/docs/md028.md) to reflect and explain this better.

That means that there are 3 options:

**Option 1:** Add the `>` marker on the blank line to make it one continuous blockquote:
```markdown
> Hello.
>
> Goodbye.
```

**Option 2:** Add separating text to clarify they're distinct blockquotes:
```markdown
> Hello.

And then:

> Goodbye.
```

**Option 3:** Disable MD028 if you prefer allowing this pattern:
```bash
rumdl check --disable MD028 yourfile.md
```
(Or of course, disable it in the config or with inline comments.)

**Comment by serl** [comment]
Hello, just adding a little use case we had, which might me interesting (or not). Note that disabling the rule is what we did and we're totally fine with that:

```markdown
> [!TIP]
> Here's a github tip

> [!NOTE]
> Here's a github note
```

They must be kept separated, otherwise the first will "eat" the second:

> [!TIP]
> Here's a github tip

> [!NOTE]
> Here's a github note

```markdown
> [!TIP]
> This tip eats the note below
> [!NOTE]
> What am I becoming?
```

> [!TIP]
> This tip eats the note below
> [!NOTE]
> What am I becoming?


(image FTR)

<img width="258" height="297" alt="Image" src="https://github.com/user-attachments/assets/75790a3c-49ac-483e-9e28-86dd1b47f8ec" />
